posthog/migrations/0156_insight_short_id.py

```diff
@@ -24,9 +24,7 @@ class Migration(migrations.Migration):
         ),
         migrations.RunPython(create_short_ids, migrations.RunPython.noop),
         migrations.AlterField(
-            model_name="dashboarditem",
-            name="short_id",
-            field=models.CharField(blank=True, max_length=12, default=posthog.models.dashboard_item.generate_short_id),
+            model_name="dashboarditem", name="short_id", field=models.CharField(blank=True, max_length=12),
```

Comment:
Is the default not needed in the migration? If that's the case, we should remove lines 26-28 altogether, we're altering the field to the same state.

Reply:
You're right, removed.

`default` in a migration should refer to a db default. In this case though the default is generated by the code when python tries to commit the model.

Reply:
I think Django still expects this. Django thinks a migration is outstanding I believe?

Reply:
Paolo is right, Django needs this as even though this default is not applied to the DB, model changes in migrations in the end must always resolve to the model's current state